### PR TITLE
[fs-detectors] Add dark/light turborepo logo

### PR DESCRIPTION
**Pull request by chloetedder**
#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by cb1kenobi — APPROVED** [review]

**Review by tknickman — APPROVED** [review]
Looks good!

**Review by styfle — CHANGES_REQUESTED** [review]
Please use `darkModeLogo` to match the Framework interface

File: packages/fs-detectors/src/monorepos/monorepo-managers.ts [hunk 1]
```diff
@@ -13,13 +13,17 @@ import type { Framework } from '@vercel/frameworks';
  *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
  *
  */
-export const monorepoManagers: Array<
-  Omit<Framework, 'description' | 'getOutputDirName'>
-> = [
+export type MonorepoManager = Omit<
+  Framework,
+  'description' | 'getOutputDirName'
+> & { darkLogo?: string };
```
Comment (styfle) [inline packages/fs-detectors/src/monorepos/monorepo-managers.ts]:
```suggestion
>
```

File: packages/fs-detectors/src/monorepos/monorepo-managers.ts [hunk 2]
```diff
@@ -13,13 +13,17 @@ import type { Framework } from '@vercel/frameworks';
...
+export const monorepoManagers: Array<MonorepoManager> = [
   {
     name: 'Turborepo',
     slug: 'turbo',
     logo: 'https://api-frameworks.vercel.sh/monorepo-logos/turborepo.svg',
+    darkLogo:
```
Comment (styfle) [inline packages/fs-detectors/src/monorepos/monorepo-managers.ts]:
```suggestion
    darkModeLogo:
```

File: packages/fs-detectors/src/monorepos/monorepo-managers.ts (hunk 1 above)
Comment (chloetedder) [reply packages/fs-detectors/src/monorepos/monorepo-managers.ts]: Oh perfect...I didn't see that in there 💯

**Review by styfle — APPROVED** [review]

**Pull request merged**